InclusionBot.md

```diff
@@ -73,9 +73,11 @@ approves your changes, they will be merged and become active in the bot.
 | master/slave, master / slave                     | primary/secondary<br>primary/replica<br>leader/worker<br>main/replica                         | This term is rooted in the enslavement of human beings. We want to avoid slavery references.<br><br>See [GitHub to replace "master" with alternative term to avoid slavery references](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/) for more.                                                                                                            |
 | open the kimono, open kimono                     | share information                                                                             | This term arises from the sexualization of Asian women. We want to avoid sexist, racist, and sexual references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                                                           |
 | grandfathered                                    | legacy<br>included retroactively                                                              | This term arose from voter restrictions that prevented Black people from voting. We want to avoid slavery and racist oppression references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                               |
```

Comment:
change (scope creep): Ever since I started working in a repo that auto-alphabetizes its yaml keys, I'm finding myself wishing these tables were alphabetized by "Instead of". Feel free to ignore that for this PR, I just needed to say it.

Reply:
Definitely ignoring for this PR, but if you wanted to reorder the markdown file, I don't think that'd be a problem! Then locally, we'd delete the yaml file and have it regenerated from the markdown. We'd need to manually re-add the exceptions from the diff, but it should be pretty minor.